python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_multioutput_firesatellite.py

```diff
@@ -1,22 +1,23 @@
 """
-Example of multi output Kriging on the fire satellite model
-===========================================================
+Example of multi output Gaussian Process Fitter on the fire satellite model
+===========================================================================
 """
 
 # %%
-# This example aims to illustrate Kriging metamodel with several outputs on the fire satellite model.
+# This example aims to illustrate Gaussian Process Fitter (Kriging) metamodel with several outputs on the fire satellite model.
 
 
 # %%
 # Loading of the model
 # --------------------
 # This model involves 9 input variables and 3 output variables.
-# We load the :ref:`Fire satellite use case<use-case-fire-satellite>`.
+# We load the :ref:`Fire satellite use case<use-case-firesatellite>`.
 
 # %%
 import openturns as ot
-from openturns.usecases.fire_satellite import FireSatelliteModel
+from openturns.usecases.fireSatellite_function import FireSatelliteModel
```

Comment:
seems thats the old name, maybe your branch is out of sync ?

Reply:
Yes. I've made a rebase to fix this point.